package.json

```diff
@@ -85,7 +85,7 @@
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
       "pre-push": "yarn build-tests-changed && yarn split-e2e-tests",
-      "pre-commit": "./scripts/package-json-check.sh && yarn verify-commit"
+      "pre-commit": "yarn prettier-changes && yarn lint-fix && ./scripts/package-json-check.sh && yarn verify-commit"
```

Comment:
use `yarn prettier-changes && yarn lint-fix` in `pre-commit` so that we don't need `lint-staged`